CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/NumberInputPane.java

```diff
@@ -63,8 +65,34 @@
     private static final Logger LOGGER = Logger.getLogger(NumberInputPane.class.getName());
 
     private static final String INVALID_ID = "invalid";
+    private String parameterId;
 
     public NumberInputPane(final PluginParameter<?> parameter) {
+
+        parameterId = parameter.getId();
+        List<String> numberRecentValues = RecentParameterValues.getRecentValues(parameterId);
+        if (numberRecentValues != null) {
+            switch (parameter.getType().getId()) {
+                case FloatParameterType.ID:
+                    if (numberRecentValues.size() > 1) {
+                        parameter.setFloatValue(Float.valueOf(numberRecentValues.get(1)));
+
+                    } else {
+                        parameter.setFloatValue(Float.valueOf(numberRecentValues.get(0)));
+                    }
+                    break;
+                case IntegerParameterType.ID:
+                    if (numberRecentValues.size() > 1) {
+                        parameter.setIntegerValue(Integer.valueOf(numberRecentValues.get(1)));
+                    } else {
+                        parameter.setIntegerValue(Integer.valueOf(numberRecentValues.get(0)));
+                    }
```

Comment:
if-else can be converted to tertiary (similar to suggestion for `ColorInputPane`)